sqlite-db/src/lib.rs

```diff
@@ -638,13 +639,11 @@ async fn load_cfd_events(
         }
     }
 
-    let mut events = events
+    let events = events
         .into_iter()
         .map(|(_, event)| event)
         .collect::<Vec<CfdEvent>>();
 
-    events.sort_unstable_by(CfdEvent::chronologically);
```

Comment:
The reason why this was introduced was because of the introduction of `into_par_iter`. We were not sure if `into_par_iter` would keep the ordering and hence ordered it again. 

Maybe it's better to also remove `into_par_iter` as well and take into account the non-parallel iter.

Reply:
Thanks for that, I actually had forgotten about his :/ 
Given the recent database changes I think we can actually consider removing the parallel iterator. Converting this to draft until then.

Reply:
I think so too: let's remove the `into_par_iter` and keep the sql sort you added in this commit. That should guarantee us the ordering.